### DISCUSSION TITLE
Configurer VSCode pour Python comme un pro | Macktireh ABDI SOUBANEH

### DISCUSSION THREAD

**Discussion by giscus** [Macktireh/macktireh.dev#47] [Blog Posts Comments]
# Configurer VSCode pour Python comme un pro | Macktireh ABDI SOUBANEH

Découvrez comment transformer VSCode en un IDE Python puissant : extensions essentielles, personnalisation, raccourcis et astuces pour coder comme un pro.

https://macktireh.dev/fr/posts/setting-up-vscode-for-python-like-a-pro/

**Comment by haabdisoub** [comment]
Excellent Merci
